errors.go

```diff
@@ -101,20 +101,35 @@ func NewExitError(message interface{}, exitCode int) ExitCoder {
 // by overiding the ExitErrHandler function on an App or the package-global
 // OsExiter function.
 func Exit(message interface{}, exitCode int) ExitCoder {
+	var err error
+
+	switch e := message.(type) {
+	case ErrorFormatter:
+		err = fmt.Errorf("%+v", message)
+	case error:
+		err = e
+	default:
+		err = fmt.Errorf("%+v", message)
+	}
```

Comment:
> Well ErrorFormatter implies that the error message can be formatted dfferently than a "regular" error.

Buf the handling of ErrorFormatter same as default

Reply:
Actually collapsing ErrorFormatter into default doesnt seem to work. I'm going to leave it as it is.